.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,6 +6,6 @@ contact_links:
   - name: 📦 Aniyomi extensions
     url: https://github.com/jmir1/aniyomi-extensions
```

Comment:
On Tachiyomi's issue page they take us to their extension list on their website (https://tachiyomi.org/extensions/ ), should I change this link so it will also redirect to Aniyomi's website extension page (https://aniyomi.jmir.xyz/extensions/) instead of Aniyomi-extensions repository on GitHub?

Reply:
yeah you can do that 👍

Reply:
Ok